CHANGELOG.md

```diff
@@ -79,7 +79,11 @@ https://github.com/googleapis/google-cloud-cpp/issues/8234.
 
 **Other Changes**
 
-* fix: treat `kDeadlineExceeded` as permanent error in all services ([#8525](https://github.com/googleapis/google-cloud-cpp/pull/8525))
+* fix: treat `kDeadlineExceeded` as permanent error in all services except ([#8525](https://github.com/googleapis/google-cloud-cpp/pull/8525)):
+  * BigQuery
+  * Bigtable Admin
+  * IAM
+  * Spanner Admin
```

Comment:
All of these services were affected by the PR: https://github.com/googleapis/google-cloud-cpp/pull/8525/commits/7cd327f8ed4d789b991a00a7c459585f6d36c407

I would put it back to the way it was one commit ago.

Reply:
Sorry about that, Scott.  Perhaps, then, we should be citing two PRs?

Reply:
changed back